js/modules/k6/timers/timers.go

```diff
@@ -126,6 +128,15 @@ func (e *Timers) timerInitialization(
 		timeout = 0
 	}
 
+	name := setTimeoutName
+	if repeat {
+		name = setIntervalName
+	}
+
+	if callback == nil {
```

Comment:
Should we be even more defensive and assert `callback` is actually "callable"?
What if the value is not callable? Would it be handled as a JS error/exception, or as an _uncaptured_ Go panic? 🤔

Reply:
My understanding is that it's either nil or callable since we already use here sobek.Callable type. For instance, sobek.AssertCallable works with Dobel.Value only.

Reply:
Ah yeah, right, my fault! Apologies! 🙏🏻